Documentation/workflow/DevelopmentTips.md

```diff
@@ -373,3 +373,69 @@ steps rely on having the Android NDK installed.
         (gdb) set sysroot /tmp/gdb-symbols
         (gdb) set solib-search-path /tmp/gdb-symbols
         (gdb) target remote :50999
+
+# .NET 6 Tips
+
+## Finding Mono runtime packs
+
+`main` builds of the Mono runtime packs are on the following NuGet
+feed, such as this `nuget.config`:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
+
+Internal MS users can view packages on Azure DevOps here:
+
+* https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet6
+
+To find the commit of a given package version, locate the `.nuspec`
```

Comment:
I don't understand how to get from the package listing above to the `.nuspec` file "inside the package".  Does the package need to be Downloaded first?

Reply:
I wrote a few more sentences on this.